appUITests/Screens/Authentication/LoginScreen.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  UIConstants.swift
+//  GoldenSampleUITests
+//
+//  Created by Backbase on 15/12/2023.
+//
+
+import Foundation
+import XCTest
+
+class LoginScreen : BaseScreen {
+    
+    //MARK: ELEMENTS
+    private lazy var titleLbl = app.staticTexts.matching(identifier: "register.titleLabel").firstMatch
```

Comment:
i agree this is a nice improvement to prevent regression failure, however, this won't work for Productised apps or model bank, since the identifier belongs to the journey level and any changes that happen in a separate project won't be automatically applied to the uitest in non-journey apps project(PA/MB).

for this golden sample,  i will put it in a specific constant for any locator has an identifier